_includes/header.html

```diff
@@ -6,6 +6,9 @@
                     <div class="col-md-8 col-md-offset-2">
                         <h1 class="brand-heading">Konstructs</h1>
                         <p class="intro-text">An open source Infiniminer/Minecraft inspired multiplayer game.</p>
+                        <p>
+                          <iframe width="280" height="158" src="https://www.youtube.com/embed/8HmZ1Rk0Kak" frameborder="0" allowfullscreen></iframe>
```

Comment:
The static size 280x158px will be problematic on smaller screens (cell phones).
Maybe something like `style="max-width: 280px; height: 158px"` ... or some thing similar.